From 98fe85548a8acd4cb9dfc53ea85fcd94fff9a745 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Wed, 29 Feb 2012 10:37:47 +0000
Subject: [PATCH] build: fix for automake 1.11.2 and newer

From automake 1.11.2 pkglib_{DATA,SCRIPTS,PROGRAMS} becomes invalid
and automake will exit with an error.

We change to pkgdata_{DATA,SCRIPTS} and pkglibexec_PROGRAMS.

The automake commit that introduces it:
http://git.savannah.gnu.org/cgit/automake.git/commit/?h=maint&id=9ca632642b006ac6b0fc4ce0ae5b34023faa8cbf

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 Makefile.am                  |    8 ++++----
 gentoo/Makefile-files        |    2 +-
 scripts/Makefile-files       |    4 ++--
 src/Makefile-files           |    4 ++--
 vserver-start/Makefile-files |    2 +-
 5 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index b9587b0..e67c1e0 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -34,9 +34,9 @@ lib_LIBRARIES =
 lib_LTLIBRARIES =
 pkglib_LTLIBRARIES =
 
-pkglib_SCRIPTS =
-pkglib_DATA =		FEATURES.txt
-pkglib_PROGRAMS =
+pkgdata_SCRIPTS =
+pkgdata_DATA =		FEATURES.txt
+pkglibexec_PROGRAMS =
 legacy_SCRIPTS =
 legacy_PROGRAMS =
 
@@ -167,7 +167,7 @@ pkgstaterevdir =	$(confdefaultsdir)/run.rev
 vserverpkgdir =		$(vserverdir)/.pkg
 vshelperstatedir =	$(localstatedir)/run/vshelper
 
-fix_SCRPTS		=  $(addprefix $(pkglibdir)/, $(notdir $(pkglib_SCRIPTS))) \
+fix_SCRPTS		=  $(addprefix $(pkglibdir)/, $(notdir $(pkgdata_SCRIPTS))) \
 			   $(addprefix $(legacydir)/, $(notdir $(legacy_SCRIPTS))) \
 			   $(addprefix $(sbindir)/,   $(notdir $(sbin_SCRIPTS))) \
 			   $(addprefix $(initrddir)/, $(notdir $(initrd_SCRIPTS)))
diff --git a/gentoo/Makefile-files b/gentoo/Makefile-files
index faf7cbd..f9abd05 100644
--- a/gentoo/Makefile-files
+++ b/gentoo/Makefile-files
@@ -29,5 +29,5 @@ EXTRA_DIST +=		$(gentoo_src_SCRPTS) \
 
 if HAVE_GENTOO_INIT
 initrd_SCRIPTS +=	$(gentoo_src_SCRPTS)
-pkglib_SCRIPTS +=	gentoo/bash-wrapper
+pkgdata_SCRIPTS +=	gentoo/bash-wrapper
 endif
diff --git a/scripts/Makefile-files b/scripts/Makefile-files
index 4422172..9958710 100644
--- a/scripts/Makefile-files
+++ b/scripts/Makefile-files
@@ -116,9 +116,9 @@ scripts_subst_SRCS =		scripts/util-vserver-vars.pathsubst \
 sbin_SCRIPTS +=			$(scripts_sbin_src_PRGS) $(scripts_sbin_gen_PRGS) \
 				$(scripts_legacy_src_PRGS)
 scripts_sbincfg_DTA =		$(scripts_sbincfg_gen_DTA)
-pkglib_SCRIPTS +=		$(scripts_pkglib_src_SCRPTS)
+pkgdata_SCRIPTS +=		$(scripts_pkglib_src_SCRPTS)
 legacy_SCRIPTS +=		$(scripts_legacy_src_SCRPTS) $(scripts_legacy_gen_SCRPTS)
-pkglib_DATA +=			$(scripts_pkglib_src_DTA) $(scripts_pkglib_gen_DTA)
+pkgdata_DATA +=			$(scripts_pkglib_src_DTA) $(scripts_pkglib_gen_DTA)
 
 CLEANFILES +=			$(scripts_pkglib_gen_DTA) \
 				$(scripts_sbin_gen_PRGS) \
diff --git a/src/Makefile-files b/src/Makefile-files
index a9a18b6..8d986a7 100644
--- a/src/Makefile-files
+++ b/src/Makefile-files
@@ -102,7 +102,7 @@ noinst_HEADERS +=	src/capability-compat.h \
 			src/context-sync.hc \
 			src/attribute-util.h
 
-pkglib_PROGRAMS +=	src/capchroot \
+pkglibexec_PROGRAMS +=	src/capchroot \
 			src/chain-echo \
 			src/chcontext-compat \
 			src/check-unixfile \
@@ -128,7 +128,7 @@ pkglib_PROGRAMS +=	src/capchroot \
 
 if ENSC_HAVE_C99_COMPILER
 if ENSC_HAVE_CRYPTO
-pkglib_PROGRAMS +=	src/vhashify
+pkglibexec_PROGRAMS +=	src/vhashify
 endif ENSC_HAVE_CRYPTO
 endif ENSC_HAVE_C99_COMPILER
 
